terraform/environments/data-platform/iam.tf

```diff
@@ -129,17 +144,6 @@ data "aws_iam_policy_document" "athena_load_lambda_function_policy" {
       "${module.s3_athena_query_results_bucket.bucket.arn}/*"
     ]
   }
-
-  statement {
-    sid = "AthenaQueryAccess"
-    actions = [
-      "athena:StartQueryExecution",
-      "athena:GetQueryExecution"
```

Comment:
the athena load lambda now has GetQueryResults access. But having one policy object for the athena access seems like a good idea.